otx/algorithms/detection/adapters/mmdet/dataset/mmdataset.py

```diff
@@ -0,0 +1,238 @@
+"""Base MMDataset for Detection Task."""
```

Comment:
I'd like to suggest to change this file name to otx_det_dataset.py. What do you think?

Reply:
I took a look at `anomalib `to see along with other comments below.
This has been modified to `mmdet/data/dataset.py`.